src/main/java/org/bigredbands/mb/views/ProjectView.java

```diff
@@ -1141,7 +1141,7 @@ public void actionPerformed(ActionEvent arg0) {
         window.addKeyListener(new HotKey());
 
         // wizard
-        Wizard myWizard = new Wizard(controller);
+        Wizard myWizard = new Wizard(controller, loaded);
```

Comment:
Rather than pass this boolean down into `Wizard`, why not just skip creating the wizard altogether based on the boolean? E.g., if you use something like `showWizard` (which is the inverse of what you have for `loaded`, i.e. `showWizard == !loaded`):

```suggestion
        if (showWizard)
            new Wizard(controller);
```

You also don't need to assign `myWizard`, since it's not used outside of `actionPerformed()`.

Reply:
Thank you for this feedback it's very helpful! I'll make these changes right away.